docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,302 @@
+
+  depends_on = [
+    helm_release.install_liqo_milan
+  ]
+
+  provider = liqo.milan
+
+}
+```
+
+### Liqo peering resource
+
+Once `generate_resource` is created, Terraform will continue peering procedure with the output parameters of the previous resource.\
+The resource in charge of doing this is `liqo_peering` of [liqo](https://google.com) provider configured with config file or parameters of local cluster:
+
+```
+provider "liqo" {
+  alias = "rome"
+  kubernetes = {
+    config_path = kind_cluster.rome.kubeconfig_path
+  }
+}
+
+resource "liqo_peering" "peering" {
+
+  depends_on = [
+    helm_release.install_liqo_rome
+  ]
+
+  provider = liqo.rome
+
+  cluster_id      = liqo_generate.generate.cluster_id
+  cluster_name    = liqo_generate.generate.cluster_name
+  cluster_authurl = liqo_generate.generate.auth_ep
+  cluster_token   = liqo_generate.generate.local_token
+
+}
+```
+
+You can check the peering status by running:
+
+```bash
+kubectl get foreignclusters
+```
+
+The output should look like the following, indicating that the cross-cluster network tunnel has been established, and an outgoing peering is currently active (i.e., the _Rome_ cluster can offload workloads to the _Milan_ one, but not vice versa):
+
+```text
+NAME    TYPE        OUTGOING PEERING   INCOMING PEERING   NETWORKING    AUTHENTICATION   AGE
+milan   OutOfBand   Established        None               Established   Established      12s
+```
+
+At the same time, you should see a virtual node (`liqo-milan`) in addition to your physical nodes:
+
+```bash
+kubectl get nodes
+```
+
+```text
+NAME                 STATUS   ROLES                  AGE     VERSION
+liqo-milan           Ready    agent                  14s     v1.23.6
+rome-control-plane   Ready    control-plane,master   7m56s   v1.23.6
+rome-worker          Ready    <none>                 7m25s   v1.23.6
+```
+
+### Liqo offload resource
+
+If you want to deploy an application that is scheduled onto Liqo virtual nodes, you should first create a namespace where your pod will be started.\
+This can be achieved with `kubernetes_namespace` of [kubernetes](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs) provider configured with config file or parameters of local cluster.
+
+Then tell Liqo to make this namespace eligible for the pod offloading.\
+The resource in charge of doing this is `liqo_offload` of the same [liqo](https://google.com) provider of `liqo_peering` resource:
+
+```
+resource "liqo_offload" "offload" {
+
+  depends_on = [
+    helm_release.install_liqo_rome,
+    kubernetes_namespace.namespace
+  ]
+
+  provider = liqo.rome
+
+  namespace = "liqo-demo"
+
+}
+```
+
+```{admonition} Note
+The virtual nodes have a [taint](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) that prevents the pods from being scheduled on them.
+The Liqo webhook will add the toleration for this taint to the pods created in the liqo-enabled namespaces.
```

Comment:
Add something like "so that those pods..."
(the sentence looks half the story to me: it says that it's adding a toleration, but it doesn't say to do what

Reply:
This section is removed following Marco Iorio's suggestions.

Reply:
Personally, I still feel this comment is not clear. I would suggest to be more precise, hence avoiding that the reader has to be be 100% aligned with all the Liqo documentation in order to understand what's written here.